src/Driver/IBMDB2/Statement.php

```diff
@@ -108,6 +108,10 @@ public function bindParam($param, &$variable, $type = ParameterType::STRING, $le
                 $this->lobs[$param] = &$variable;
                 break;
 
+            case ParameterType::BINARY:
+                $this->bind($param, $variable, DB2_PARAM_IN, DB2_BINARY);
+                break;
+
```

Comment:
Your feature should not require a change in the drivers because array parameter expansion happens before the statement hits the driver. If binary parameters didn't need special treatment before, they shouldn't now.